<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/assets/favicon.ico</normaloff>:/icons/assets/favicon.ico</iconset>
  </property>
  <widget class="ScaledBgWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 12pt &quot;Minecraft&quot;;</string>
   </property>
   <widget class="QPushButton" name="saveAndCloseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/assets/bgbtn.png);</string>
    </property>
    <property name="text">
     <string>Save and close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel keys:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>230</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border: solid white 2px;
font: 500 15pt &quot;Minecraft&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="muteCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>230</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	spacing: 15px;
}

QCheckBox::indicator {
	width: 32px;
	height: 32px;
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: solid white 2px;
}

QCheckBox::indicator:checked {
	width: 32px;
	height: 32px;
	image: url(:/images/assets/checkmark.png);
}

QCheckBox::indicator:unchecked {
	width: 32px;
	height: 32px;
}</string>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledBgWidget</class>
   <extends>QWidget</extends>
   <header>scaledbgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
